service/security/authentication/api/src/main/java/org/eclipse/kapua/service/authentication/mfa/MfaAuthenticationService.java

```diff
@@ -0,0 +1,60 @@
+/*******************************************************************************
+ * Copyright (c) 2020 Eurotech and/or its affiliates and others
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     Eurotech - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.kapua.service.authentication.mfa;
+
+import org.eclipse.kapua.service.KapuaService;
+
+import java.util.List;
+
+/**
+ * MfaAuthenticationService interface
+ */
+public interface MfaAuthenticationService extends KapuaService {
+
+    /**
+     * @return true if the MfaAuthenticationService is enabled, false otherwise
+     */
+    boolean isEnabled();
+
+    /**
+     * Validates a code generated with the Google Authenticator app
```

Comment:
Remove Google Authenticator from the Java doc

Reply:
Ok, this will be done in PR #3115 .